graviti/dataframe/frame.py

```diff
@@ -142,6 +129,19 @@ def _construct(
         obj._index = index
         return obj
 
+    @classmethod
+    def from_lazylists(cls: Type[_T], lazylists: Dict[str, Any]) -> _T:
```

Comment:
why we need this classmethod? seems this method is just a redirect of __init__?

Reply:
yes, just wanted to formally distinguish between the two ways of creating DataFrame.